packages/fluentui/react-northstar/src/components/Dropdown/Dropdown.tsx

```diff
@@ -573,7 +573,7 @@ export const Dropdown = (React.forwardRef<HTMLDivElement, DropdownProps>((props,
         {createShorthand(Button, triggerButton, {
           defaultProps: () => ({
             className: dropdownSlotClassNames.triggerButton,
-            content,
+            content: { content, id: triggerButtonContentId },
```

Comment:
Just FYI: We don't merge `defaultProps` & user's input deeply, if someone will specify `content` i.e. `<Dropdown triggerButton={{ content: 'Foo }} />` this `id` will be gone. But now sure how it's important.

Reply:
I also thought, but if you look at line 553, it simplifies things:

```
const content = getSelectedItemAsString(value[0]);
```

so in this case content is always a string.